Update office hours time to account for DST

mikesplain opened the pull request:
We didn't move office hours to account for DST even though UTC doesn't change. 

Maybe it's easiest to change UTC until the end of DST in the US?

mikesplain commented:
/area documentation

chrislovecnm commented:
Should we just list UTC?  What do the sigs do?

andrewsykim commented:
It might actually make sense to not list UTC at all. If you list only EST/PST, it's likely people will look up the time zone conversion on google where it will account for DST.

mikesplain commented:
@chrislovecnm @andrewsykim Comments addressed. I dropped UTC and cleaned it up to be similar with [the community docs](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle).

andrewsykim commented:
/lgtm

andrewsykim commented:
/test pull-kops-e2e-kubernetes-aws
